test/spec/struct.wast

```diff
@@ -105,3 +105,29 @@
   )
   "invalid rtt"
 )
+
+(assert_invalid
+  (module
+    (type $vec (struct (field (ref func))))
```

Comment:
Would it be better to make this field `i32`, if we are using `i32`s for tests below? So that we can only test the number of fields, not the type of fields.

Reply:
Good point, will fix.